.github/workflows/release.yml

```
@@ -0,0 +1,315 @@
+              <ul>
+                  <li><a href="core/index.html">Core Module</a> - Query building and execution</li>
+                  <li><a href="cache/index.html">Cache Module</a> - Query caching strategies</li>
+                  <li><a href="optimizer/index.html">Optimizer Module</a> - Query optimization</li>
+              </ul>
+              <p><a href="https://github.com/query4j/dynamicquerybuilder">Back to GitHub Repository</a></p>
+          </body>
+          </html>
+          EOF
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.7)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)